### PR TITLE
Add key pair change tests to E2E.Test

### PR DESCRIPTION
**Pull request by exfinen**
- Refactor existing E2E.Test code to reuse for the new tests
- Remove unused code and variables in E2E.Test
- Add test cases for key pari changes to E2E.Test

### CODE REVIEW

File: packages/contracts/test/E2E.Test.ts [hunk 1]
```diff
@@ -177,81 +142,246 @@ contract('E2E', (accounts) => {
...
+            await letAllVotersVote(voterKeypairs, voiceCredit_1, 1, undefined)
+            await timeTravel2EndOfVotingPeriod()
+            await tally()
+            const tallyResult = await getTallyResult()
 
+            // coordinator withdraws deposits and transfer them to each recipient
             for (let i = 0; i < RECIPIENTS.length; i++) {
-                // transfer tokens voted to recipient to recipient
-                const counts = resultsArr[i]
+                // coordintor transfer tokens voted to recipient currently owned by cream to recipient
```
Comment (kazuakiishiguro) [inline packages/contracts/test/E2E.Test.ts:383]: @exfinen 
typo: coordintor->coordinator

**Comment by kazuakiishiguro** [comment]
Found a minor typo at https://github.com/couger-inc/cream/pull/109/files#r752195863

**Review by kazuakiishiguro — CHANGES_REQUESTED** [review]
Found a minor typo at https://github.com/couger-inc/cream/pull/109/files#r752195863

File: packages/contracts/test/E2E.Test.ts (hunk 1 above)
Comment (exfinen) [reply packages/contracts/test/E2E.Test.ts:383]: no objection!

Comment (exfinen) [reply packages/contracts/test/E2E.Test.ts]: fixed

**Review by kazuakiishiguro — APPROVED** [review]

**Pull request merged**